src/backends/github/__tests__/API.spec.js

```diff
@@ -0,0 +1,39 @@
+import AssetProxy from "../../../valueObjects/AssetProxy";
```

Comment:
Fantastic - mocking the GH API is an existing issue (https://github.com/netlify/netlify-cms/issues/330). This is a great step towards that, which will let us add a test suite for the editorial workflow.